### ISSUE TITLE
Latest Version on NPM

### ISSUE THREAD

**Issue by bloigge** [jieter/leaflet-headless#15]
I think there is a minor version conflict with your release on NPM. The setSize function doesn't include the resetView.

````
       setSize: function (width, height) {
            this._size = new L.Point(width, height);
            // reset pixelOrigin
            this._resetView(this.getCenter(), this.getZoom());
            return this;
        }
````

**Comment by jieter** [comment]
released v0.1.2 from current master.

**Comment by bloigge** [comment]
thx
